swagger/Platfrom API/bizaiChats.json

```diff
@@ -0,0 +1,407 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "title": "Platform Management",
+    "description": "",
+    "contact": {},
+    "version": "1.0.0"
+  },
+  "servers": [
+    {
+      "url": "https://api.vcita.biz/v3/"
+    }
+  ],
+  "tags": [
+    {
+      "name": "chatmng",
```

Comment:
You need to have 2 tags BizAIChat and BizAIChatMessage and add them correspondingly to the appropriate endpoints

Reply:
The tags will become the name of the API group that are tagged together. any endpoint with the the same tag will be grouped and named according to its tag. 
<img width="315" alt="image" src="https://github.com/vcita/developers-hub/assets/151043415/580fea2f-3438-48cc-8b62-51cba356a866">
in this example you see the developer used "card request" and "cards as tags. We want to group messages CRUD + List and to group Chat CRUD +List